src/webnn_native/ops/LeakyReluOperator.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef WEBNN_NATIVE_OPS_LEAKYRELUOPERATOR_H_
+#define WEBNN_NATIVE_OPS_LEAKYRELUOPERATOR_H_
+
+#include "webnn_native/Graph.h"
+#include "webnn_native/Operator.h"
+
+namespace webnn_native { namespace op {
+
+    class LeakyReluOperator final : public OperatorBase {
```

Comment:
Could `LeakyReluOperator` share implementation `LeakyRelu`? They share same code such as alpha setting etc.,.

Reply:
Thanks! I will try to do.